.github/workflows/ci-multiarch.yaml

```diff
@@ -111,5 +111,5 @@ jobs:
             echo "[!] The job 'build-images' didn't provide any outputs. Can't create the manifest list."
             exit 1;
           fi
-          docker manifest create ${{ env.IMAGE }}:multiarch-next $AMEND
-          docker manifest push ${{ env.IMAGE }}:multiarch-next
+          docker manifest create ${{ env.IMAGE }}:next $AMEND
+          docker manifest push ${{ env.IMAGE }}:next
```

Comment:
Is it OK for default next tag to have arch optionally?

Hard to tell possible cause but GitHub action can push, let's say, only arm64 image...
My point is if multiarch is pushed to default, then we must guarantee that every platform is build before push, or at least `amd64` which is dependency for most cases, which is not the case currently.

Reply:
yes, we should be able to quietly warn if s390x fails, but loudly error if any other arch is missing.

Reply:
> Nobody complained about removing multi-arch build

true... but there's no operator CSV which looks for `:multiarch-next` tags so there's no obvious/easy way to even use these multiarch images. If we use :next then at least the nightly CSV would refer to them and we'd get community testing / PR checks / HP tests etc.

Reply:
Then let's just leave multiarch

Reply:
does that mean "keep the :multiarch-next` tag? 

or does that mean "default to `:next` being multiarch so we immediately expose problems as they happen, and actually use multiarch-built-bits for bleeding edge testing?

Reply:
I meant

> "default to :next being multiarch so we immediately expose problems as they happen, and actually use multiarch-built-bits for bleeding edge testing?